### ISSUE TITLE
Runtime issue: TypeError: Cannot read property 'filter' of undefined

### ISSUE THREAD

**Issue by LEvisC** [dequelabs/axe-core#2516]
I have a problem that accessibility score in Lighthouse is ?. But problem occur only in mobile test, on desktop all errors in section Background and foreground colors have a sufficient contrast ratio are solved. So the test for desktop is 85 %.
And the test in mobile mode is ? only on home section on other subpages is OK.

How to solve the problem?

```
TypeError: Cannot read property 'filter' of undefined
    at T (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:432085)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:433053
    at Array.map (<anonymous>)
    at Object.i.getTextElementStack (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:433030)
    at Object.a.getRectStack (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:424621)
    at Object.a.filteredRectStack (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:424170)
    at Object.a.getBackgroundStack (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:423573)
    at Object.a.getBackgroundColor (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:423075)
    at Object.evaluate (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:345078)
    at k.run (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:46:152463)
```

**Comment by straker** [comment]
Thanks for the issue. Can you provide the URL to the page that is giving the error? Otherwise my only guess is it might be a similar problem to #2483

**Comment by LEvisC** [comment]
The page is fortech.si. It is very strange because on time accessibility score in Lighthouse is ? for desktop and some times is OK and it shows a score. For mobile as a wrote score is always ? ( error).

Thank you.

**Comment by WilcoFiers** [comment]
We believe this issue was resolved with #2619. If you find this again in axe-core 4.1, please open another issue. The difference was likely because of different screen widths.

**Comment by kmajay978** [comment]
**###**
